rdl/rdl-gen-athenz-server/main.go

```diff
@@ -407,8 +408,9 @@ func (gen *javaServerGenerator) resourcePath(r *rdl.Resource) string {
 
 func (gen *javaServerGenerator) handlerBody(r *rdl.Resource) string {
 	noContent := r.Expected == "NO_CONTENT" && r.Alternatives == nil
-	s := "        try {\n"
-	s += "            ResourceContext context = this.delegate.newResourceContext(this.request, this.response);\n"
+	s := "        int code = ResourceException.OK;\n"
+	s += "        ResourceContext context = this.delegate.newResourceContext(this.request, this.response);\n"
```

Comment:
Not sure we can move this call out of try block. If it throws an exception, it might expose our trace and some other internal data to the caller which we can't have. We should be able to define the ResourceContext context = null outside of the try block and keep the delegate call within the block and within the recordMetrics calls make sure to handle the case where context is null.